docs/courses/README.md

```diff
@@ -43,11 +43,12 @@ You'll need to have an idea on basic terminology used in LibreLingo:
 - A **Course** is a set of Modules. There is typically one course for each language pair. e.g. Spanish for English speakers, Japanese for Italian speakers, etc.
 - A **Module** is a set of Skills.
 - A **Skill** is a small unit of course material. In the web app, the user will typically practice a Skill at a time.
+- A **Challenge** is a single question or task the user will complete within a skill.
```

Comment:
hmm, I think Course/Module/Skill *vs* Challenge are not really comparable as they are not the same category.

Course/Module/Skill are categories within the YAML based course system. Challenges are a different concept that are specific to the web app.

The distinction is that the YAML format is meant to be a generic format for building courses, whereas challenges live in the JSON format which is a different layer that is used to implement the web app. This might still be relevant to course authors, but the fact that there's no such thing as "Challenge" in the YAML files is important.

(The other changes in this file make sense)

Reply:
Yeah, you're right. I wanted to define the word Challenge somewhere, but this isn't a good place. I'll try a different way, and we can see if that's better.